.github/workflows/tests.yaml

```diff
@@ -349,23 +350,24 @@ jobs:
                      --bridge-app ./out/darwin-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
                   "
 
-            - name: Run Tests using the python parser sending commands to chip-tool
-              run: |
-                  ./scripts/run_in_build_env.sh \
-                  "./scripts/tests/run_test_suite.py \
-                     --runner chip_tool_python \
-                     --chip-tool ./out/darwin-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
-                     --target-skip-glob '{Test_TC_DGTHREAD_2_1,Test_TC_DGTHREAD_2_2,Test_TC_DGTHREAD_2_3,Test_TC_DGTHREAD_2_4}' \
-                     run \
-                     --iterations 1 \
-                     --test-timeout-seconds 120 \
-                     --all-clusters-app ./out/darwin-x64-all-clusters-${BUILD_VARIANT}/chip-all-clusters-app \
-                     --lock-app ./out/darwin-x64-lock-${BUILD_VARIANT}/chip-lock-app \
-                     --ota-provider-app ./out/darwin-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
-                     --ota-requestor-app ./out/darwin-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
-                     --tv-app ./out/darwin-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/darwin-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                  "
+            #   https://github.com/project-chip/connectedhomeip/issues/27673
+            # - name: Run Tests using the python parser sending commands to chip-tool
+            #   run: |
```

Comment:
If we just want to disable this for the tsan variant (which is what seems to be failing), how about:

1. We move the `if: matrix.build_variant != 'no-ble-tsan-clang'` to this job.
2. We remove it from the "Run Tests" immediately above.

Alternately, `if: false` or something might be better in terms of disabling, but then we should still remove the bit from the job above, or disable the tsan variant altogether, so we don't waste time on it if it's not running any tests.

Reply:
Done